integrated_channels/sap_success_factors/models.py

```diff
@@ -117,14 +119,17 @@ def get_learner_data_record(self, enterprise_enrollment, completed_date=None, gr
             completed_timestamp = parse_datetime_to_epoch(completed_date)
             course_completed = is_passing
 
-        return LearnerDataTransmissionAudit(
-            enterprise_course_enrollment_id=enterprise_enrollment.id,
-            sapsf_user_id=enterprise_enrollment.enterprise_customer_user.get_remote_id(),
-            course_id=enterprise_enrollment.course_id,
-            course_completed=course_completed,
-            completed_timestamp=completed_timestamp,
-            grade=grade,
-        )
+        sapsf_user_id = enterprise_enrollment.enterprise_customer_user.get_remote_id()
```

Comment:
I would still like there to be a log message that says we weren't able to find the remote id - I think it will be helpful for @mattdrayer when setting up and testing SAP with customers

Reply:
I agree with @brittneyexline -- can't hurt to write a log entry -- let's be sure we indicate the proper log level so the alerts don't become noisy.